packages/osd-monaco/src/xjson/language.ts

```diff
@@ -48,8 +50,11 @@ registerLexerRules(monaco);
 // @ts-ignore
 window.MonacoEnvironment = {
   getWorker: (module: string, languageId: string) => {
-    if (languageId === ID) {
-      // In OpenSearch Dashboards we will probably build this once and then load with raw-loader
+    // In OpenSearch Dashboards we will probably build this once and then load with raw-loader
+    if (languageId === 'json') {
+      const blob = new Blob([jsonWorkerSrc], { type: 'application/javascript' });
+      return new Worker(URL.createObjectURL(blob));
+    } else if (languageId === ID) {
```

Comment:
I'm not sure it makes sense for this logic to be in the `xjosn/language.ts` file. Shouldn't we have a similar `json/language.ts` file instead?

Reply:
It's a todo written in comment. The `getWorker` method is a global singleton and we can not rewrite the method in json/language.ts. The `getWorker` method need a refractor indeed.

Reply:
Yeah I agree that `window.MonacoEnvironment` should be a global var which call `getWorker` to check worker path, then load it. Is there a tracking issue or plan to work on this?

Reply:
Meanwhile, I think if you gonna modify this, then we should also work on constants.ts which defines ID. Currently, it only has xjson 
```
export const ID = 'xjson';
```
which doesn't make sense after you add json. Maybe we could either update it or remove it.